ports/stm32/Makefile

```diff
@@ -64,13 +64,17 @@ INC += -Ilwip_inc
 CFLAGS_CORTEX_M = -mthumb
 
 # Select hardware floating-point support
+SUPPORTS_HARDWARE_FP_SINGLE = 0
+SUPPORTS_HARDWARE_FP_DOUBLE = 0
 ifeq ($(CMSIS_MCU),$(filter $(CMSIS_MCU),STM32F765xx STM32F767xx STM32F769xx STM32H743xx))
 CFLAGS_CORTEX_M += -mfpu=fpv5-d16 -mfloat-abi=hard
+SUPPORTS_HARDWARE_FP_DOUBLE = 1
```

Comment:
Should also add `SUPPORTS_HARDWARE_FP_SINGLE = 1` in case one builds with single precision float on such an MCU

Reply:
Done